src/app/clients/ClientsPage.tsx

```diff
@@ -43,13 +45,39 @@ const fullNameSearch = (
     return query.ilike("full_name", `%${state}%`);
 };
 
+const postcodeSearch = (
+    query: PostgrestFilterBuilder<Database["public"], any, any>,
+    state: string
+): PostgrestFilterBuilder<Database["public"], any, any> => {
+    return query.ilike("address_postcode", `%${state}%`);
```

Comment:
I recommend copying and pasting the postcode search code from the parcels table. It's not quite as simple as this, because we need to be able to search for a `null` postcode in the backend when "NFA" is input into the filter, so it's not a direct mapping between the state and the value we put into the the `.ilike`.